conf/config.go

```diff
@@ -69,6 +69,8 @@ type MenderConfigFromFile struct {
 
 	// Global retry polling max interval for fetching update, authorize wait and update status
 	RetryPollIntervalSeconds int `json:",omitempty"`
+	// Global max retry poll count
+	RetryPollCount int `json:",omitempty"`
```

Comment:
@kacf do you think we should use that outside inventory too?

Reply:
Yes. Should be relatively easy if you change `GetExponentialBackoffTime` as mentioned.